.github/workflows/e2e.yaml

```diff
@@ -40,11 +40,17 @@ jobs:
       with:
         fetch-depth: 0
 
+    - uses: Kong/kong-license@7302c8154056c156d28b7eff412b966fc351ca08
```

Comment:
Not going to use "@master"

Reply:
Yeah, agreed, since it's our own action we may as well just get patches as they come, even if we can't choose a specific semver major version or whatever. If we do get something breaking, we'll notice quickly and the people to fix are nearby.

Reply:
I feel that we're making an exception here, trying to use tags (or SHAs ideally) for other action but using a branch tip for our own action 🤔 

🤷 I'm not going to push this if you feel this is "OK" to use `@master`. I'll have this comment as a safeguard that it wasn't me proposing this reckless, bleeding edge strategy 🤣

Reply:
Changed, PTAL.